src/platform/macos.rs

```diff
@@ -390,11 +390,11 @@ impl WindowBuilderExtMacOS for WindowBuilder {
 }
 
 pub trait EventLoopExtMacOS {
-  /// Sets the activation policy for the application. It is set to
+  /// Sets the initial activation policy for the application. It is set to
```

Comment:
```suggestion
  /// Sets the activation policy for the application. It is set to
```
Lets keep this as is.